encoding/masking-limiting-etc.md

```diff
@@ -138,6 +196,34 @@ see [kageru's blog post][].
 
 [kageru's blog post]: https://kageru.moe/blog/article/edgemasks
 
+I would also lump the range mask
```

Comment:
With how many different authors we have now, using `I` seems odd.
That's just for futore thought, however, since you didn't change that in this PR.

Reply:
Perhaps put the author's name (Zastin, I think) in parantheses after the “I”?

Reply:
Either that, or rephrase to be less opinionated and more generally applicable, I say.

Reply:
I changed around the wording on all the uses of `I`, let me know what you guys think.